semperfiwebdesign/all-in-one-seo-pack
# Fix Grunt issues in aioseop_class.php

Issue #69

## Proposed changes

Fixes Grunt issues that got through. These are just a few spacing changes.

## Types of changes

What types of changes does your code introduce?

- Improves existing code

## Checklist


## Testing instructions

Install and open the General Settings menu. and check that there aren't any notices. Also, @wpsmort 
 when you test this have a quick look at the code as well.
